<enhancement>(administration config): New --skip-healthchecks option for vector validate (#19691)
vectordotdev/vector: website/content/en/docs/administration/validating.md

vector validate --no-environment /etc/vector/vector.yaml
```

#### Skipping health checks

To validate the vector configuration even if the health-checked endpoints are not reachable

> (i.e. from a local workstation), but still run all other environment checks, use the

Reviewer: ```suggestion
(for example, from a local workstation), but still run all the other environment checks, use the
```

Author: ...and here.

[`--skip-healthchecks`][skip_healthchecks] flag:

```bash
vector validate --skip-healthchecks /etc/vector/vector.yaml
```

> Note that the configured `data_dir` must still be writeable.

Reviewer: ```suggestion
**Note:** The configured `data_dir` must still be writeable.
```

Author: Thans, I changed it here..
